addons/mail/static/src/discuss/core/public/welcome_page.js

```diff
@@ -43,10 +64,18 @@ export class WelcomePage extends Component {
         this.props.proceed?.();
     }
 
+    get canJoin() {
+        return this.state.userName.trim() && this.state.userName.length <= 60;
+    }
```

Comment:
Would impose the `<= 60` limit for custom name. If the name comes from `this.store.self_user?.name`, we should have this limitation as nothing prevents having such as long name on the `res.user`'s name.

Reply:
But you made the right change I see, thanks ;)